rosa_sts_managed_oidc_with_cognito_idp/cognito.tf

```diff
@@ -13,13 +13,13 @@ resource "aws_cognito_user_pool_domain" "domain" {
 
 resource "aws_cognito_user" "user" {
   user_pool_id = aws_cognito_user_pool.cluster-idp.id
-  username     = "rosa-admin"
-  password     = var.rosa_password
+  username     = var.admin_username
```

Comment:
Is this the same admin creds as the cluster, or should it be different? Just thinking in terms of does it need to be the same to work our would a separate user be more appropriate?

Reply:
It's the same thing, just using Cognito instead of htpasswd. I was trying to achieve consistence between plans